tools/project_manager/update_illos.php

```diff
@@ -0,0 +1,206 @@
+
+$operation_image_str;
+if ($operation == 'replace') {
+    $operation_image_str = _('Replace Illustration');
+} else {
+    $operation_image_str = _('Delete Illustration');
+}
+
+output_header("$operation_image_str: {$project->nameofwork}");
+
+echo "<h1>" . html_safe($project->nameofwork) . "</h1>\n";
+echo "<h2>$operation_image_str: $image</h2>\n";
+
+if (!$project->can_be_managed_by_current_user)
+{
+    echo "<p>", _('You are not authorized to manage this project.'), "</p>\n";
+    return;
+}
+
+if ($operation == 'delete' && $project->state != PROJ_NEW)
+{
+    echo "<p>", _('You can only delete illustrations for a project in the new state.'), "</p>\n";
+    return;
+}
+
+$page_image_names = array();
+$res = mysqli_query(DPDatabase::get_connection(), "
+    SELECT image
+    FROM $projectid
+    ORDER BY image
+") or die(mysqli_error(DPDatabase::get_connection()));
+while (list($page_image) = mysqli_fetch_row($res))
+{
+    $page_image_names[] = $page_image;
+}
+
+chdir("$projects_dir/$projectid");
+$existing_image_names = glob("*.{png,jpg}", GLOB_BRACE);
+// That returns a sorted list of the .png files
+// followed by a sorted list of the .jpg files,
+// but we want the two lists interleaved...
+$nonpage_image_names = array_diff($existing_image_names, $page_image_names);
+if (!in_array($image, $nonpage_image_names) || !is_file("$projects_dir/$projectid/$image"))
+{
+    // This too should only happen if someone is URL-tweaking.
+    // (Or conceivably, the file was recently deleted, and the user
+    // has an image_index that was generated before the deletion.)
+    echo "<p>", _('There is no such illustration in the project.'), "</p>\n";
+    return;
+}
+
+if ( isset($_FILES['replacement_image']) )
+{
+    // The user has uploaded a file.
+    $err_msg = handle_upload($projectid, $image, $_FILES['replacement_image'] );
+    $success_msg = _('Illustration successfully replaced.');
+} elseif (@$_REQUEST['confirmed'] == 'yes') {
+    // The user has uploaded a file.
+    $err_msg = handle_delete($projectid, $image);
+    $success_msg = _('Illustration successfully deleted.');
+}
+
+if (isset($success_msg)) {
+    if ( $err_msg == '' )
+    {
+        echo "<p>", $success_msg, "</p>\n";
+        echo "<p>",
+            "<a href='$code_url/tools/proofers/images_index.php?project=$projectid'>",
+            _('Return to Images Index'),
+            "</a>",
+            "</p>\n";
+        exit;
+    }
+    else
+    {
+        echo "<p class='error'>";
+        echo _('An error occurred.'), "\n";
+        echo $err_msg, "\n";
+        echo _('Please try again.'), "\n";
+        echo "</p>\n";
+        // Fall through to try again.
+    }
+}
+
+if ($operation == 'replace') {
+    echo "<p>", _('Select a replacement illustration to upload:'), "</p>\n";
+    echo "
+        <form enctype='multipart/form-data' action='update_illos.php' method='post'>
```

Comment:
For HTML5, which we are using, you can drop the `action` attribute: `<form enctype='multipart/form-data' method='POST'>`.